examples/jupyter/post-processing.ipynb

```diff
@@ -213,18 +212,18 @@
     "particles = 5000\n",
     "\n",
     "# Instantiate a Settings object\n",
-    "settings_file = openmc.Settings()\n",
-    "settings_file.batches = batches\n",
-    "settings_file.inactive = inactive\n",
-    "settings_file.particles = particles\n",
+    "settings = openmc.Settings()\n",
+    "settings.batches = batches\n",
+    "settings.inactive = inactive\n",
+    "settings.particles = particles\n",
```

Comment:
Let's get rid of the temporary variables above:
```Python
settings.batches = 100
...
```

Reply:
Thanks for the feedback @paulromano, I'll get cracking.